pkg/cpio/fs_unix.go

```diff
@@ -6,6 +6,7 @@ package cpio
 
 import (
 	"fmt"
+	"golang.org/x/sys/unix"
```

Comment:
nit: The style is to put the non-standard packages in their own sections. For an example, see: https://github.com/u-root/u-root/blob/master/pkg/kexec/kexec_linux_amd64.go#L12

Reply:
darn it. I did not even pay attention, I just assumed gofmt did this and did not look. Fixing now.

Reply:
Although in many ways calling this a non-standard package is a bet of a stretch (to me).

Reply:
Fixed, can you take a quick look, since the one line fix reset approved :-)